runtime/dir.go

```diff
@@ -19,6 +19,14 @@ import (
 	"path/filepath"
 )
 
+// LogsDir returns the default directory for Service Weaver logs,
+// /tmp/serviceweaver/logs. We recommend that deployers store their logs in a
```

Comment:
My understanding is that os.TempDir() isn't always /tmp/. Can you clarify it here?